File: ProblemB/ProblemB.c
#include <stdio.h>
#include <math.h>

void Fractal(int n,int level)
{
	int p=pow(3,n);

	if(level==1)
	{
		printf("FD 1/%d\n",p);
		printf("LD 60\n");
		printf("FD 1/%d\n",p);
		printf("LD 240\n");
		printf("FD 1/%d\n",p);
		printf("LD 60\n");
		printf("FD 1/%d\n",p);
	}
	else
	{
		Fractal(n,level-1);
		printf("LD 60\n");
		Fractal(n,level-1);
		printf("LD 240\n");
		Fractal(n,level-1);
		printf("LD 60\n");
		Fractal(n,level-1);
	}
}

void output(int n)
{
	Fractal(n,n);
}

int main(void)
{
	int n;
	scanf("%d",&n);
	output(n);
	return 0;
}

File: ProblemB/ProblemB.py
def Fractal(order,level):
	if(level==1):
		p=3**order
		print("FD 1/%d"%p)
		print("LD 60")
		print("FD 1/%d"%p)
		print("LD 240")
		print("FD 1/%d"%p)
		print("LD 60")
		print("FD 1/%d"%p)
	else:
		Fractal(order,level-1)
		print("LD 60")
		Fractal(order,level-1)
		print("LD 240")
		Fractal(order,level-1)
		print("LD 60")
		Fractal(order,level-1)

def output(n):
	Fractal(n,n)

N=int(input())
output(N)